crates/opencascade/src/law_function.rs

```diff
@@ -0,0 +1,22 @@
+use cxx::UniquePtr;
+use glam::dvec2;
+use opencascade_sys::ffi;
+
+use crate::primitives::make_point2d;
+
+#[must_use]
+pub(crate) fn law_function_from_graph(
```

Comment:
I thought that exposing `LawFunction` on the public API would be premature at this point.

Reply:
Agreed, let's keep it local to the crate for now.